### ISSUE TITLE
Related Blog QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#822]
Few changes to these blocks:
- [ ] Change the background color from yellow to hex #ECECEC
- [ ] change title color to hex #F06D2F
- [ ] Change hover/focus state of the author to just underline, no color change.
- [ ] Any chance we could top align the post title and have the teaser just come beneath it with consistent padding?
- [ ] Keep the author in the same place
<img width="1239" alt="Screen Shot 2020-05-05 at 12 00 23 PM" src="https://user-images.githubusercontent.com/21184833/81105502-e2ece200-8ec8-11ea-960c-d371f3dbea5e.png">

**Comment by JulesKhong** [comment]
For the cards, all of the text is read as the link and it's confusing on a screenreader and probably not that helpful for SEO. 

I really like the BBC's approach to this. Make the Heading the link. Then at the end of the card add an anchor tag with the card link as the href, absolutely position it over the whole card and give it `tabindex='-1' aria-hidden='true'` What this does is it makes the whole card clickable with the hidden link, but a user tabbing through only has the heading links showing up so the link purposes are really clear. 
Here's a screenshot of a card on bbc.com's homepage. You can use z-index to keep other links on top of the invisible one. 
<img width="958" alt="Screen Shot 2020-05-05 at 7 39 50 PM" src="https://user-images.githubusercontent.com/19519395/81134016-36caeb80-8f08-11ea-8fc9-35d8625bb1b0.png">

**Comment by komejo** [comment]
Done and ready for QA on https://a11y-2020--thinkshout.netlify.app/blog/2020/03/critical-digital-focus-covid/

@JulesKhong I realized that just linking the titles was fine? I think linking the whole post may have been some HTML5 exuberance. (We can wrap everything in a link now, yaaay!)

**Comment by vickibrown** [comment]
@komejo just a couple things
- [ ] The authors need links just like they have everywhere else.
- [ ] The post titles should be hex #F06D2F

**Comment by komejo** [comment]
Done and ready for final QA on https://a11y-2020--thinkshout.netlify.app/blog/2020/03/critical-digital-focus-covid/

**Comment by JulesKhong** [comment]
- [ ] Now that author names are links in the cards, they should have a visible focus and hover state 
<img width="417" alt="Screen Shot 2020-06-01 at 1 40 05 PM" src="https://user-images.githubusercontent.com/19519395/83452397-7acce580-a40d-11ea-8621-8ffa36a8b1ff.png">

**Comment by komejo** [comment]
Fixed!

**Comment by vickibrown** [comment]
@komejo looks good!
